src/sbp_fileio.c

```diff
@@ -81,25 +83,27 @@ static void read_cb(u16 sender_id, u8 len, u8 msg[], void* context)
   len += cfs_read(f, buf + len, readlen);
```

Comment:
Ah, the `u8` len + the `u8` readlen in the call to `cfs_read` could overwrite buf 256 bytes. Added MIN(readlen, 255-len) to determine how much to read into the buffer. Thanks!